scripts/spi_flash/fatfs_api.h

```diff
@@ -26,7 +26,7 @@ struct ff_file_info {
     uint16_t modified_date;
     uint16_t modified_time;
     uint8_t  attrs;
-    char     name[13];
+    char     name[256];
```

Comment:
This change is not enough to match the `FILINFO` struct in the fatfs library:
```c
typedef struct {
    FSIZE_t	fsize;			/* File size */
    WORD	fdate;			/* Modified date */
    WORD	ftime;			/* Modified time */
    BYTE	fattrib;		/* File attribute */
#if FF_USE_LFN
    TCHAR	altname[FF_SFN_BUF + 1];/* Altenative file name */
    TCHAR	fname[FF_LFN_BUF + 1];	/* Primary file name */
#else
    TCHAR	fname[12 + 1];	/* File name */
#endif
} FILINFO;
```
This results in a `malloc(): invalid size (unsorted) ` error when the cffi wrapper attempts to `memcpy` the contents of a `FILINFO` struct to the `ff_file_info` struct in the `fatfs_get_fstats()` function .  To resolve this you will need to add an `char altname[13]` field to the `ff_file_info` struct above the `name` field.